docs/enterprise-guide/authenticating-with-saml.md

```diff
@@ -115,14 +115,16 @@ Metabase will now need to know some things about your IdP. Here's a breakdown of
 
 ### Settings for signing SSO requests (optional)
 
-These are additional settings you can fill in to sign SSO requests to ensure they don’t get tampered with.
+These are additional settings you can fill in to sign SSO requests to ensure they don’t get tampered with. In addition, if your IdP encrypts SAML responses, you'll need to ensure this section is filled out.
 
 **SAML keystore path:** the absolute path to the keystore file to use for signing SAML requests.
 
 **SAML keystore password:** if it wasn't already self-evident, this is just the password for opening the keystore.
 
 **SAML keystore alias:** the alias for the key that Metabase should use for signing SAML requests.
 
+**Important note:** If you change any of these settings, you will need to restart Metabase due to the way the keystore file is read.
```

Comment:
I'd maybe put this above the list of settings. And stylistically, I think it would be better to italicize the `Important note` text so that it has a different treatment than the names of the settings.

Also, do they have to restart when setting these things _initially_, or only after _editing_ an existing value?

Reply:
*nod* re: agree placement and styling of `Important note`.

Need to double-check re: initial setting vs. editing. I _think_ it is either but will confirm.

Reply:
Confirmed that a restart is required in both cases.